instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -347,6 +355,29 @@ def _instrument(self, **kwargs):
                 unit="bytes",
             )
 
+        if "process.runtime.thread_count" in self._config:
+            self._meter.create_observable_gauge(
+                name="process.runtime.thread_count",
```

Comment:
Please follow the semantic conventions for [process](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/process-metrics.md#process) and [runtime](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/runtime-environment-metrics.md#runtime-environment-specific-metrics---processruntimeenvironment) specific instruments.

Reply:
Since it was fixed in [3.0.1](https://github.com/wiremind/remoulade/issues/339#issuecomment-1735019094) we can just ignore dependency version 3.0.0